### PR TITLE
TASK-59: fix merge conflicts #closed.

### PR DESCRIPTION
**Pull request by cham207388**
Fix merge conflict issues

### CODE REVIEW

File: src/components/Main/About/About.js [hunk 1]
```diff
@@ -19,7 +16,7 @@ const About = () => {
                         <div className="card-content">
                             <div className="content">
                                 <h4>Our Mission</h4>
-                                <p>To connect the immigrant communities together and share their unique stories for inspiration and awareness.</p>
```
Comment (jjblues86) [inline src/components/Main/About/About.js]: I made an edit to the sentence.

File: src/components/Main/About/About.js [hunk 2]
```diff
@@ -19,7 +16,7 @@ const About = () => {
...
+                                <p>To connect the immigrant communities together and share their unique stories for inspiration and awareness on a broader scale.</p>
```
Comment (jjblues86) [inline src/components/Main/About/About.js]: Let's keep the current sentence and remove "on a broader scale"

File: src/components/Main/About/About.js [hunk 3]
```diff
@@ -51,8 +48,7 @@ const About = () => {
             </div>
             <br/>
             <div className="has-text-centered">
-            <h1 className={styles.Stories}>Creators</h1>
-                <hr className={styles.Divider}/>
+                <h3 className="title is-2 Blue">Creators</h3>
```
Comment (jjblues86) [inline src/components/Main/About/About.js:51]: This is a different blue compared to our logo

File: src/components/Main/Stories/Stories.js [hunk 4]
```diff
@@ -53,11 +52,9 @@ class Stories extends Component {
         return (
 
             <div>
-                <div className={styles.storiesDivider}>
-                    <h1 className={styles.Stories}>Immigrant Stories</h1>
-                    <hr className={styles.Divider}/>
-                </div>
-                <div className={styles.cardsList}>
+                <div className={style.StoriesContainer}>
+                    <h1 className={style.StoriesHeader}>The Immigrant Stories</h1>
```
Comment (jjblues86) [inline src/components/Main/Stories/Stories.js:56]: Let's keep this instead 
```
      <div className={styles.storiesDivider}>
                    <h1 className={styles.Stories}>Immigrant Stories</h1>
                    <hr className={styles.Divider}/>
                </div>
```

**Review by jjblues86 — CHANGES_REQUESTED** [review]
Made some comments

**Comment by cham207388** [comment]
Removed "on a broader scale" as you suggested.

**Pull request merged**